src/docker-osx-dev

```diff
@@ -389,7 +389,8 @@ function init_boot2docker {
 # Inspect especific configuration about the docker-machine
 #
 function inspect_docker_machine {
-  docker-machine inspect --format="$*" "$DOCKER_MACHINE_NAME" 2>&1
+  local result=$(docker-machine inspect --format="$*" "$DOCKER_MACHINE_NAME" 2>&1)
+  echo ${result/<no value>//}
```

Comment:
I just want to make sure I'm understanding this correctly. Is the idea here that `docker-machine` (or rather, the go template engine) will return `<no value>` when a value isn't found, so when you look up the two keys, `.StorePath` and `.HostOptions.AuthOptions.StorePath`, one will return a value, one will return `<no value>`, and this line here will remove the <no value>? 

If so, is it possible for both values to be present? Or neither?